packages/functional-tests/pages/relier.ts

```diff
@@ -70,6 +70,7 @@ export class RelierPage extends BaseLayout {
   }
 
   async promptNoneError() {
+    await this.page.waitForSelector('.error');
```

Comment:
The [docs](https://pptr.dev/api/puppeteer.page.waitforselector) indicate that `page.waitForSelector` returns an element for the provided selector. Is there a reason not to use this return value? It seems like the call to `page.locator` is likely redundant.

Reply:
Yes, fixed it! Thanks!